client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -304,6 +341,34 @@ private void write01(ReaderGroupConfig object, RevisionDataOutput revisionDataOu
             revisionDataOutput.writeMap(object.endingStreamCuts, keySerializer, valueSerializer);
             revisionDataOutput.writeInt(object.getMaxOutstandingCheckpointRequest());
         }
+
+        private void read02(RevisionDataInput revisionDataInput, ReaderGroupConfigBuilder builder) throws IOException {
```

Comment:
I've made changes to the new revision as described in the serializer framework. Only the new values are serialized instead of the whole object in order to mitigate the redundancy mentioned.

Reply:
I do not see this change.